bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/config/CameraConfig.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ipcamera.config;
+
+import java.math.BigDecimal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link CameraConfig} handles the configuration of cameras.
+ *
+ * @author Matthew Skinner - Initial contribution
+ */
+@NonNullByDefault
+public class CameraConfig {
+    String ipAddress = "";
+    BigDecimal port = new BigDecimal(0);
```

Comment:
Why not use `int`?
```suggestion
    int port;
```
Default is 0, so no `= 0` needed.